lib/rubocop/cop/rails/find_by.rb

```diff
@@ -45,6 +46,15 @@ def autocorrect(node)
             corrector.replace(first_loc, '')
           end
         end
+
+        private
+
+        def where_with_method_as_param?(node)
+          where = node.children.find { |el| el.method?(:where) }
+          return unless where
+
+          where.arguments.size == 1 && where.arguments.first.send_type?
```

Comment:
Hmm, I think string literal also should be added an offense if it can be used as `find_by`'s argument. But I can't find a difference between `find_by` and `where.take`, so, I can not tell if this change is correct :thinking:  https://github.com/bbatsov/rubocop/issues/4096#issuecomment-284925700